### PR TITLE
Django webauth

### PR DESCRIPTION
**Pull request by JobDoesburg**
Closes #ISSUE

### Summary
A clear and concise description of the changes that you made. What bug did you solve? Or what feature did you add?

### How to test
Steps to test the changes you made:

### CODE REVIEW

File: website/thaliawebsite/settings.py [hunk 1]
```diff
@@ -604,6 +605,23 @@ def from_env(
...
+    development="localhost:8000",
+    production="thalia.nu",
+    staging="staging.thalia.nu",
+)
+WEBAUTH_ORIGIN = setting(
+    development="http://localhost:8000",
+    production="https://thalia.nu",
+    staging="https://staging.thalia.nu",
+)
+WEBAUTH_VERIFY_URL = "/user/webauth/verify/"
```
Comment (se-bastiaan) [inline website/thaliawebsite/settings.py:623]: This could reuse the SITE_DOMAIN setting

**Comment by se-bastiaan** [comment]
This doesn't support TOTP right? And how would this integrate with the API?

**Comment by JobDoesburg** [comment]
> This doesn't support TOTP right? 
Yes, this works solely via FIDO. I did just find https://github.com/jazzband/django-two-factor-auth which looks promising too (it's from jazzband) so I might investigate this too

> And how would this integrate with the API?
For the API, one could say that 2FA is also required to authorize OAuth applications with certain requirements, and then the OAuth API key becomes a second factor in itself (as it is stored on the device).

**Comment by se-bastiaan** [comment]
Yes I already mentioned that lib in the issue :)

If the 2fa can be done at login then yes it works perfect with oauth, but it needs to do that obviously

**Comment by JobDoesburg** [comment]
https://www.calazan.com/how-to-enable-2fa-in-the-django-admin/

**Comment by DeD1rk** [comment]
django-webauth calls itself not-production-ready and is not being maintained.
https://github.com/jazzband/django-two-factor-auth has become the standard package to use.

**Pull request closed without merging**